### DISCUSSION TITLE
LogLevel for Kubernetes via Fluentbit

### DISCUSSION THREAD

**Discussion by davehamer** [datalust/seq-tickets#1308] [Help]
Hi All,

Quick question - is there a way to log the correct loglevel for each event when logging Kubernetes cluster events via Fluentbit?

They're all coming through as "info" at the moment which is defined in the upstream helm chart.

TIA

**Comment by nblumhardt** [comment]
Hi Dave,

By cluster events, do you mean events from K8s itself, or `STDOUT`/`STDERR` routed from applications? If the latter, logging them in newline-delimited JSON with an `@l` level property should do the job. (Less sure about K8s events themselves.)

Good chance I'm misunderstanding you, here, so any config/sample events/etc. that you can send to help pain the picture would be handy.
